src/xrc/xh_bttn.cpp

```diff
@@ -55,6 +77,17 @@ wxObject *wxButtonXmlHandler::DoCreateResource()
 
     SetupWindow(button);
 
+    SetBitmapIfSpecified(button, &wxBitmapButton::SetBitmapPressed,
```

Comment:
This looks like too a copy-pasto (and explains at least some of the errors in the CI builds):
```suggestion
    SetBitmapIfSpecified(button, &wxButton::SetBitmapPressed,
```
(same thing below too, of course).